frontend/src/components/Wheel/index.js

```diff
@@ -1,89 +1,72 @@
-/*
-import React from "react";
+import { useEffect, useState } from "react";
 import SpinButton from "./button";
 import "./wheel.css";
-import WheelComponent from "react-wheel-of-prizes";
 
-const CreateSVG = () => {
-  return (
-    <PieCircle
-      slices={[
-        { percentage: "0.10", color: "blue" },
-        { percentage: "0.20", color: "green" },
-        { percentage: "0.25", color: "red" },
-        { percentage: "0.30", color: "yellow" },
-      ]}
-    />
-  );
-};
-
-export const PieCircle = (props) => {
-  let totalPercentage = 0;
+const Wheel = () => {
+  const [wheelInfo, setWheelInfo] = useState([]);
+  const [mainColor, setMainColor] = useState("");
+  const [secondaryColor, setSecondaryColor] = useState("");
 
-  const getCoordinatesForPercent = () => {
-    const x = Math.cos(2 * Math.PI * totalPercentage);
-    const y = Math.sin(2 * Math.PI * totalPercentage);
-    return [x, y];
-  };
-
-  const _renderSlice = ({ percentage, color }) => {
-    const [startX, startY] = getCoordinatesForPercent();
-    totalPercentage += percentage;
-    const [endX, endY] = getCoordinatesForPercent();
-
-    const largeArcFlag = percentage > 0.5 ? 1 : 0;
-    const pathData = [
-      `M ${startX} ${startY}`, // Move
-      `A 1 1 0 ${largeArcFlag} 1 ${endX} ${endY}`, // Arc
-      "L 0 0", // Line
-    ].join(" ");
-
-    return <path d={pathData} fill={color} key={pathData} />;
-  };
+  useEffect(() => {
+    fetch(`${process.env.REACT_APP_BACKEND_PATH}/wheel`)
+      .then((response) => {
+        if (response.ok) {
+          return response.json();
+        } else {
+          throw response;
+        }
+      })
+      .then((data) => {
+        setWheelInfo(data)
+        setMainColor(data.config.main_color)
+        setSecondaryColor(data.config.secondary_color)
+      });
+  }, []);
 
   return (
-    <svg
-      viewBox="-1 -1 2 2"
-      style={{ transform: [{ rotate: "-90deg" }] }}
-      height="100"
-      width="100"
-    >
-      {props.slices.map((slice) => _renderSlice(slice))}
-    </svg>
-  );
-};
-
-const wheel = () => {
-  const name = "Wheel";
+    <div>
+      <table className="table">
+        <tr>
+          <th>ID</th>
+          <th>Name</th>
+          <th>Logo</th>
+          <th>Prizes</th>
+          <th>Color1</th>
+          <th>Color2</th>
+        </tr>
 
-  const prizes = ["Prize1", "Prize2", "Prize3", "Prize4"];
+        {/* {wheelInfo.map((wheel) => {
+          return ( */}
```

Comment:
Please remove the comments in this file, unless they're really necessary

Reply:
Done